rules/web/web_java_in_access_log.yml

```diff
@@ -0,0 +1,30 @@
+title: Java Payload String
+id: 583aa0a2-30b1-4d62-8bf3-ab73689efe6c
+status: experimental
+description: Detects possible Java payload in web access logs
+author: frack113
+date: 2022/06/04
+references:
+    - https://www.rapid7.com/blog/post/2022/06/02/active-exploitation-of-confluence-cve-2022-26134/
+    - https://www.rapid7.com/blog/post/2021/09/02/active-exploitation-of-confluence-server-cve-2021-26084/
+    - https://github.com/httpvoid/writeups/blob/main/Confluence-RCE.md
+    - https://twitter.com/httpvoid0x2f/status/1532924261035384832
+logsource:
+    category: webserver
+detection:
+    keywords:
+      - '%24%7B%28%23a%3D%40'
+      - '${(#a=@'
+      - '%24%7B%40java'
+      - '${@java'
+      - '\u0022java'
+      - '"java'
```

Comment:
Don't you think this one is prone to FPs?

Reply:
Replace '\u0022java' and '"java'  by `'java.lang.'` ?
In the writeups find java.lang.ProcessBuilder and  java.lang.Runtime

Reply:
I'd say, only the `"java` string could be problematic